packages/components/pages/f-account-info/test-utils/component-objects/f-account-info.component.js

```diff
@@ -0,0 +1,134 @@
+    get changePasswordButton () { return $(CHANGE_PASSWORD_BUTTON); }
+    get deleteAccountLink () { return $(DELETE_ACCOUNT_LINK); }
+
+    fields = {
+        firstName: {
+            get input () { return $(FIELDS.firstName.input); },
+            get emptyError () { return $(FIELDS.firstName.emptyError); },
+            get invalidError () { return $(FIELDS.firstName.invalidError); }
+        },
+        lastName: {
+            get input () { return $(FIELDS.lastName.input); },
+            get emptyError () { return $(FIELDS.lastName.emptyError); },
+            get invalidError () { return $(FIELDS.lastName.invalidError); }
+        },
+        emailAddress: {
+            get input () { return $(FIELDS.emailAddress.input); }
+        },
+        phoneNumber: {
+            get input () { return $(FIELDS.phoneNumber.input); },
+            get emptyError () { return $(FIELDS.phoneNumber.emptyError); },
+            get invalidError () { return $(FIELDS.phoneNumber.invalidError); }
+        },
+        addressLine1: {
+            get input () { return $(FIELDS.addressLine1.input); },
+            get emptyError () { return $(FIELDS.addressLine1.emptyError); }
+        },
+        addressLine2: {
+            get input () { return $(FIELDS.addressLine2.input); }
+        },
+        addressLine3: {
+            get input () { return $(FIELDS.addressLine3.input); }
+        },
+        city: {
+            get input () { return $(FIELDS.city.input); },
+            get emptyError () { return $(FIELDS.city.emptyError); }
+        },
+        postcode: {
+            get input () { return $(FIELDS.postcode.input); },
+            get emptyError () { return $(FIELDS.postcode.emptyError); },
+            get invalidError () { return $(FIELDS.postcode.invalidError); }
+        }
+    }
+
+    load () {
+        super.load(this.component);
+    }
+
+    click (field) {
+        this.fields[field].input.click();
+    }
+
+    waitForComponent () {
+        super.waitForComponent(this.component);
+    }
+
+    isComponentDisplayed () {
+        return this.component.isDisplayed();
+    }
+
+    /**
+    * @description
+    * Select all the text in the field and then performs a backspace to clear the field
+    *
+    * @param {String} fieldName The name of the field input it is clearing
+    */
+    clearBlurField (fieldName) {
+        // Determines the OS
+        const CONTROL = process.platform === 'darwin' ? 'Command' : '\uE009';
+        const el = this.fields[fieldName].input;
+        el.click();
+        el.keys([CONTROL, 'a']);
+        el.keys(['Backspace']);
+    }
+
+    /**
+    * @description
+    * Inputs customer details into the account-info component.
+    *
+    * @param {Object} customerInput customer input details
+    */
+    populateAccountForm (field, customerInput) {
+        this.fields[field].input.setValue(customerInput[field].input);
+    }
+
+    clickOutOfInputField () {
```

Comment:
Purpose of this? Is this because errors only appear on blur of fields?

Reply:
yep it is indeed :)